src/status_im/contexts/settings/saved_addresses/view.cljs

```diff
@@ -0,0 +1,41 @@
+(ns status-im.contexts.settings.saved-addresses.view
+  (:require [quo.core :as quo]
+            [quo.theme :as quo.theme]
+            [react-native.core :as rn]
+            [react-native.safe-area :as safe-area]
+            [status-im.common.resources :as resources]
+            [status-im.contexts.settings.saved-addresses.style :as style]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(defn empty-state
+  []
+  (let [theme (quo.theme/use-theme)]
+    [quo/empty-state
+     {:title           (i18n/label :t/no-saved-addresses)
+      :description     (i18n/label :t/you-like-to-type-43-characters)
+      :image           (resources/get-themed-image :sweating-man theme)
+      :container-style style/empty-container-style}]))
+
+(defn view
+  []
+  (let [inset-top           (safe-area/get-top)
+        customization-color (rf/sub [:profile/customization-color])
+        saved-addresses     []]
```

Comment:
Is this like a placeholder till we get the actually date from app-db?

Reply:
Yes. I didn't want to add the re-frame sub yet because it introduces QA surface area.